python/ray/rllib/evaluation/keras_policy_graph.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from keras import backend as K
+import numpy as np
+from ray.rllib.evaluation.policy_graph import PolicyGraph
+
+
+def _sample(probs):
+    return [np.random.choice(len(pr), p=pr) for pr in probs]
+
+
+class KerasPolicyGraph(PolicyGraph):
```

Comment:
Should probably add a disclaimer that this is half-baked compared to the TF / Torch graphs.

Reply:
Done.